scripts/common/yaml.sh

```diff
@@ -41,3 +41,25 @@ function setup_kubeadm_kustomize() {
     rm -rf $DIR/kustomize/kubeadm/join-patches
     mkdir -p $DIR/kustomize/kubeadm/join-patches
 }
+
+function setup_installer_crd() {
+    CREATE_INSTALLER_CRD_YAML="/tmp/installer_crd.yaml"
+    INSTALLER_BASE_YAML_FILE="/tmp/kurl_installer.yaml"
+
+    echo $INSTALLER_CRD | base64 --decode > $CREATE_INSTALLER_CRD_YAML
+
+    kubectl apply -f $CREATE_INSTALLER_CRD_YAML
+
+    cat > $INSTALLER_BASE_YAML_FILE << EOF
+$INSTALLER_YAML
+EOF
+
+    sed -i 's/kurl.sh/cluster.kurl.sh/g' $INSTALLER_BASE_YAML_FILE
```

Comment:
What's this for?

Reply:
I see, it's the API group. Can we update the Installer to have this group?

Reply:
For now the controller-gen code requires a group while the existing kurl infra does not have one. So the CRD is cluster.kurl.sh and the yaml from the api is kurl.sh. This fixes that pending further discussion on if we want to address this on the controller-gen side or the kurl api side.